code/modules/maptext_alerts/text_blurbs.dm

```diff
@@ -150,7 +150,7 @@ but should see their own spawn message even if the player already dropped as USC
 
 	addtimer(CALLBACK(GLOBAL_PROC, /proc/fade_blurb, targets, T), duration)
 
-/proc/fade_blurb(list/mob/targets, obj/T)
+/proc/fade_blurb(list/mob/targets, obj/T, var/fadeout_time = 0.5 SECONDS)
```

Comment:
Did this file get changed accidentally? I'm not seeing the connection.

Reply:
was part of my initial refactor attempt before I realised a port would just be better and easier